nifi-registry/nifi-registry-core/nifi-registry-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -59,4 +59,18 @@ nifi.registry.bootstrap.sensitive.key=
 nifi.registry.bootstrap.protection.hashicorp.vault.conf=./conf/bootstrap-hashicorp-vault.conf
 
 # AWS KMS Sensitive Property Providers
-nifi.registry.bootstrap.protection.aws.kms.conf=./conf/bootstrap-aws.conf
\ No newline at end of file
+nifi.registry.bootstrap.protection.aws.kms.conf=./conf/bootstrap-aws.conf
+
+# Note: the following mapping properties only apply if a Sensitive Property Provider that uses property contexts
+# is configured.  Otherwise, these values are ignored.
+#
+# If no nifi.registry.bootstrap.protection.context.mapping.* properties are provided, the context for protected
+# properties uses a 'default' context, as in "default/Manager Password".  Properties in nifi.properties are always
+# assigned this context, but there is a possibility of naming conflicts among the other configuration files.
+#
+# To create separate contexts for properties, you may provide context mappings in the format:
+# nifi.registry.bootstrap.protection.context.mapping.<contextName>=<identifier matching regex>
+# With the following configuration, for example, any property named "Manager Password" located inside
+# a block whose <identifier> starts with "ldap-" will be mapped to the context named "ldap/Manager Password",
+# regardless of whether it resides in authorizers.xml or identity-providers.xml.
+nifi.registry.bootstrap.protection.context.mapping.ldap=ldap-.*
```

Comment:
As mentioned above, it seems best to remove this section completely for the purpose of this PR.